src/index.js

```diff
@@ -0,0 +1,92 @@
+import './style.css';
+import getTasks from './getTasks.js';
+import addTask from './addTask.js';
+
+let editableID;
+let editBoleen = false;
+const submit = document.querySelector('.enter');
+const input = document.querySelector('.input');
+const btnCompleted = document.querySelector('.btnCompleted');
+const displayTodo = () => {
+  const taskContainer = document.querySelector('.task-box');
+  const todos = getTasks();
+  let li = '';
+  todos.forEach((todo, id) => {
+    li += `<li class="task">
+                <label for="${id}">
+                  <input class ="task-cls" type="checkbox" data-id = ${id}>
+                  <p>${todo.description}</p>
+                </label>
+                <div class="settings">
+                  <i class="uil uil-ellipsis-h"></i>
+                  <ul class="task-menu">
+                    <li class = "edit" data-id= "${todo.description}" id="${id}"><i class="uil uil-pen"></i>Edit</li>
+                    <li class = "delete" id="${id}"><i class="uil uil-trash-alt"></i>Delete</li>
```

Comment:
- [ ] Based on the YAGNI principle of writing clean code, I suggest you remove the 'Delete' text since there is a trash icon to listen to execute the delete functionality.

Reply:
Thank you for your feedback